bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/StringType.java

```diff
@@ -13,10 +13,14 @@
 
 public class StringType implements PrimitiveType, State, Command {
 
-    public final static StringType EMPTY = new StringType("");
+    public final static StringType EMPTY = new StringType();
 
     private final String value;
 
+    public StringType() {
+        this.value = "";

```

Comment:
For HSB and Percent type you already call the other constructor.
Perhaps we could use `this("");` here